pulp_container/app/tasks/sync_stages.py

```diff
@@ -165,8 +137,14 @@ async def get_signature_source(self):
                     list_dc = self.create_tagged_manifest_list(
                         tag_name, saved_artifact, content_data
                     )
-                    for manifest_data in content_data.get("manifests"):
-                        man_dc = self.create_manifest(list_dc, manifest_data)
+                    for listed_manifest_task in asyncio.as_completed(
+                        [
+                            self.create_listed_manifest(list_dc, manifest_data)
+                            for manifest_data in content_data.get("manifests")
+                        ]
```

Comment:
Should we also handle a state where a manifest could not be downloaded in `self.create_listed_manifest`?

Reply:
https://github.com/pulp/pulp_container/issues/648

Let's keep refactoring and features separate.